backend/modules/model_gateway/__init__.py

```diff
@@ -24,16 +26,18 @@
     and os.path.exists(MODELS_CONFIG_SAMPLE_PATH)
     and not os.path.exists(MODELS_CONFIG_PATH)
 ):
-    logger.info(
-        "models_config.json not found. Creating models_config.json from models_config.sample.json"
-    )
-    data = {
-        "provider_name": "truefoundry",
-        "api_format": "openai",
-        "llm_model_ids": ["openai-main/gpt-4-turbo", "openai-main/gpt-3-5-turbo"],
-        "embedding_model_ids": ["openai-main/text-embedding-ada-002"],
-        "api_key_env_var": "TFY_API_KEY",
-        "base_url": settings.TFY_LLM_GATEWAY_URL,
+    logger.info("models_config.yaml not found. Creating a default models_config.yaml")
+    config = {
```

Comment:
We shouldn't do this - this is very custom logic and shouldn't enter the main codebase. its better to ask the user to create a file or provide a default file.

Reply:
This is for only TF deployments.